robottelo/common/helpers.py

```diff
@@ -315,3 +315,31 @@ def patched_computeresource_init(self, server_config=None, **kwargs):
             computeresource_init(self, server_config, **kwargs)
             self._fields['url'].default = docker_url
         entities.ComputeResource.__init__ = patched_computeresource_init
+
+
+def prepare_import_data(tar_path=None):
+    """Fetch and uncompress the CSV files from the source."""
+    tmpdir = '/tmp/'
```

Comment:
The `/tmp` directory may by any application. Extracting files directly in to this directory is a Bad Thing, as you run the risk of scribbling all over the state of other applications. Also, you run the risk of using files extracted by previous calls to this function. Each call to this function should be isolated from previous and future calls to this function.

Instead of hard-coding this path here, ask the system for a directory with `mktemp -d`.